feature/visited/src/main/kotlin/uk/govuk/app/visited/data/VisitedRepo.kt

```diff
@@ -0,0 +1,12 @@
+package uk.govuk.app.visited.data
+
+import uk.govuk.app.visited.data.store.VisitedLocalDataSource
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+internal class VisitedRepo @Inject constructor(
+    localDataSource: VisitedLocalDataSource,
+) {
+    val visitedItems = localDataSource.visitedItems
```

Comment:
It would probably be better to map the realm object to a domain object to ensure encapsulation of realm within the data layer.

Reply:
Fixed up